src/components/project/dto/list-projects.dto.ts

```diff
@@ -89,9 +89,23 @@ export abstract class ProjectFilters {
   })
   readonly presetInventory?: boolean;
 
+  @Field({
```

Comment:
These have been intentionally kept internal.
Idiomatic for GQL is `partner { projects }`. This is just the implementation detail.